llmfoundry/callbacks/eval_gauntlet_callback.py

```diff
@@ -95,6 +96,14 @@ def __init__(self,
                 assert weight is not None
                 benchmark['weighting'] = weight
 
+
+        self.averages = {}
+        if averages is not None:
+            self.averages = dict(averages)
+        else:
+            # if no averages spec provided, simply average everything
+            self.averages['average'] = list(self.categories.keys())
```

Comment:
for disambiguity, should this be `global_average`? I worry about `average` being misleading, tho this is a minor point.